raiden/raiden_service.py

```diff
@@ -720,19 +720,6 @@ def leave_all_token_networks(self):
         state_change = ActionLeaveAllNetworks()
         self.handle_state_change(state_change)
 
-    def close_and_settle(self):
```

Comment:
We may have wanted to expose this again at some point but sure we can delete it. If you remove this you can also remove the `leave_all_token_networks` function right above as it's only used in this one function you just removing here.

Reply:
Since today is ur day off I implemented the suggestion and now the PR is ready to go.